src/style.css

```diff
@@ -1,3 +1,77 @@
-body {
+:root {
+  --active: #00ACDC;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header {
```

Comment:
you should use class selectors for styling instead of tag ones. check cases below